docs/buildingpex.rst

```diff
@@ -95,12 +95,12 @@ You can then import and manipulate modules like you would otherwise:
     >>> import psutil
     >>> ...
 
-Requirements can also be specified using the requirements.txt format, using ``pex -r``.  This can be a handy
+Requirements can also be specified using the requirements.txt format as generated by ``pip freeze``.  This can be a handy
```

Comment:
Hrm, this effectively wiped out a recommendation about using -r. The `pip freeze` output is a tiny subset of what can go in requirements.txt.

So, the structure before of this `Specifying requirements` section was:
1. You can specify command line requirement strings like so.
2. You can also use requirements files like so.

Now the structure is:
1. You can specify command line requirements like so.
2. A small misrepresentation of requirements.txt format / pip freeze.

How about:
1. You can specify command line requirements like so.
1a. A handy example of 1 is pex $(pip freeze) since `pip freeze` emits a list of pinned requirement strings for you like `setuptools==2.1.2` mentioned in section 1 (not these words exactly - this is just me trying to explain the structure).
2. You can also use requirements files with -r just like with Pip.

Reply:
Ah, thanks for the clarification - I see what the section was going for now. Will make a corresponding change!